src/input/password-input.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+interface Props {
+    id: string;
+    value: any;
+    onBlur?: (event: React.FormEvent<HTMLInputElement>) => void;
+    append?: string;
+    prepend?: string;
+    onChange: (value: any) => void;
+    disabled?: boolean;
+    className?: string;
+    placeholder?: string;
+    onMouseEnter?: (event: React.MouseEvent<HTMLDivElement>) => void;
+    onMouseLeave?: (event: React.MouseEvent<HTMLDivElement>) => void;
+    autoFocus?: boolean;
+    passwordVisible?: boolean;
```

Comment:
Same as line 18

Reply:
Actually in this case we need a way to initialise the `type` of the input - i.e. you want to define on init if the password field is going to be visible or not. So it needs to be configurable from outside.

Reply:
But then the naming would not make sense if you switch it to be type number or sth else. I actually like the idea of a distinct high level password component only changing the type internally

Reply:
As commented above the component is called `PasswordInput`; generally we change the `type` of the `input` to `text` just to show things. So I don't expect anyone is looking for a `number` type in a field that is used to type a password.